docs/pages/grid.md

```diff
@@ -10,7 +10,7 @@ tags:
 
 ## Basics
 
-Start by adding an element with a class of `.row`. This will create a horizontal block to contain vertical columns. Then add elements with a `.column` class within that row. You can use `.column` or `.columns`&mdash;the only difference is grammar. Specify the widths of each column with the `.small-#`, `.medium-#`, and `.large-#` classes.
+Start by adding an element with a class of `.row`. This will create a horizontal block to contain vertical columns. Then add elements with a `.column` class within that row. You can use `.column` or its <a href="http://localhost:3000/grid.html#sass-reference">alias</a> `.columns`&mdash;the only difference is grammar. Specify the widths of each column with the `.small-#`, `.medium-#`, and `.large-#` classes.
```

Comment:
6kb: what does it give in % ?

And this information can be in the sassdoc comment: it helps to choose the value of the option, and do not need examples.